src/libraries/System.ServiceProcess.ServiceController/README.md

```diff
@@ -0,0 +1,16 @@
+# System.ServiceProcess.ServiceController
+Provides the [`System.ServiceProcess.ServiceController`](https://learn.microsoft.com/en-us/dotnet/api/system.serviceprocess.servicecontroller) class, which allows you to connect to a running or stopped service, manipulate it, or get information about it.
+
+Note that `System.ServiceProcess.ServiceController` supports Windows only.
+
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/system.serviceprocess.
+
+## Contribution Bar
+- [ ] [We consider new features, new APIs and performance changes](../README.md#primary-bar)
+- [ ] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We don't accept refactoring changes due to new language features](../README.md#secondary-bars)
```

Comment:
This one's confusing too: The default secondary bar is to _not_ accept refactoring changes. So here we are just confirming that we don't accept them? Or do we accept them, contrary to the default bar?